src/lib/routes/edge-api/index.ts

```diff
@@ -59,12 +63,15 @@ export default class EdgeController extends Controller {
             middleware: [
                 this.openApiService.validPath({
                     tags: ['Edge'],
+                    summary:
+                        'Accepts a list of tokens, returns tokens that are valid and which projects they can access',
+                    description:
+                        'Send a list of tokens, only returns the valid ones',
                     operationId: 'getValidTokens',
-                    requestBody: createRequestSchema(
-                        'validateEdgeTokensSchema',
-                    ),
+                    requestBody: createRequestSchema('tokenStringListSchema'),
                     responses: {
                         200: createResponseSchema('validateEdgeTokensSchema'),
+                        ...getStandardResponses(400),
```

Comment:
Can't this return 401/403? And what about 415?

Reply:
Can't return 401/403, doesn't require a authentication header, there's no auth happening here. 415 is possible

Reply:
Updated with 6ac9cc0